src/App.js

```diff
@@ -1,16 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
+const chatLogData = chatMessages;
 
 const App = () => {
+  const [updatedChatLogData, setChangeLikeState] = useState(chatLogData)
```

Comment:
Because the function `setChangeLikeState` changes the data structure `updatedChatLogData` references, I would recommend naming the function something that reflects that action, even though in this project it's only called when we update the liked value of a message.